chromium_src/chrome/browser/ui/views/location_bar/icon_label_bubble_view.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_LOCATION_BAR_ICON_LABEL_BUBBLE_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_LOCATION_BAR_ICON_LABEL_BUBBLE_VIEW_H_
+
+#define BRAVE_GET_HIGHLIGHT_PATH_H \
```

Comment:
hmm, we're patching the .cc file but we're overriding the .h file. So I'm not sure how this works? We need to be providing the macro definition in the .cc file.

ah, it works because we're not `#undef` the macro, so it's staying around forever 😄 

So I think the code is good but we'll need to move it to `icon_label_bubble_view.cc` and call the macro `BRAVE_GET_HIGHLIGHT_PATH` (and make sure it gets undefined too).